apps/examples/testcase/Makefile

```diff
@@ -195,8 +195,9 @@ ifeq ($(CONFIG_EXAMPLES_TESTCASE_TASK_MANAGER_UTC),y)
 endif
 ifeq ($(CONFIG_EXAMPLES_TESTCASE_TASK_MANAGER_ITC),y)
 	$(Q) $(call REGISTER,taskmgr_itc,itc_taskmanager_main,TASH_EXECMD_ASYNC,100,2048)
+	$(Q) $(call REGISTER,task0,task0_main,TASH_EXECMD_ASYNC,101,1024)
```

Comment:
I don't want to show `task0` at commnad list of shell. Noone knows what it is. Even `tc` is used for Samsung certification test, because this is an opensource and we opened tc, we should concern name and etc.
Let's rename this.

Reply:
Previously task0 was tm_builtin and you commented to change it. Can you suggest a appropriate name. task0 is a builtin task(just runs in the background when started) which has dedicate permission in the testcase and when other than parent task tries to start task0, it should give No permission to start.

Reply:
@sunghan-chang please review

Reply:
@sunghan-chang can you suggest a name?